.github/workflows/test.yml

```diff
@@ -82,6 +82,8 @@ jobs:
           --env HOST_JOBS_DIR=/tmp/prairielearn \
           --name=test_container \
           prairielearn/prairielearn:${{ env.COMMIT_SHA }} /bin/bash
+        # We don't install playwright into the Docker image since it's a large
+        # dependency and we don't need it for the smoke tests.
```

Comment:
Nit, this comment doesn't seem immediately connected to its surroundings?

Reply:
Fixed in 43076c24721d5d0691338f9f2c625d28dbc46a4a.